webpack.config.js

```diff
@@ -0,0 +1,16 @@
+module.exports = {
+  entry: "./source/index.js",
```

Comment:
Yep, they exist in mine. I was just adding the webpack config premturely. Feel free to leave this PR open until the file exist (i.e. merged from another PR that closes #25)

Working on #25 currently.